src/Components/Shared/Pagination.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Pagination, PaginationItem, PaginationLink } from "reactstrap";
+
+export default class DotPagination extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+
+    this.state = {
+      activePage: 1,
+      pageSize: 0,
+      totalItems: 0
+    };
+  }
+
+  componentDidMount() {
+    this.setState(state => ({
+      ...state,
+      pageSize: this.props.sizePerPage,
+      totalItems: this.props.totalItems
+    }));
+  }
+
+  pageChange(pageNumber) {
+    this.setState(state => ({
+      ...state,
+      activePage: pageNumber + 1
+    }));
+
+    this.props.onPageChange(pageNumber);
```

Comment:
Are you sure this works as expected? because the ```activePage: pageNumber + 1``` but you're calling the `onPageChange` with `pageNumber` not `+ 1`.